common/objects/class-comment-object.php

```diff
@@ -32,16 +42,60 @@ protected function init_object( $id ) {
 
 	}
 
+	/**
+	 * Method create adds the comment object in the database.
+	 *
+	 * @param array $args accepts variables required for the comment object as array.
+	 *
+	 * @return int|WP_Error returns ID of comment or an error.
+	 *
+	 * @since 1.0.0
+	 */
 	protected function create( $args ) {
-
+		$inserted = wp_insert_comment( $args );
```

Comment:
When using a callable inside another callable, especially if it comes from WordPress or another plugin, use the `@see` tag: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-functions-class-methods